src/Nethermind/Nethermind.Api/NethermindApi.cs

```diff
@@ -208,7 +208,7 @@ public ISealEngine SealEngine
         public ITxPoolInfoProvider? TxPoolInfoProvider { get; set; }
         public IHealthHintService? HealthHintService { get; set; }
         public IRpcCapabilitiesProvider? RpcCapabilitiesProvider { get; set; }
-        public ITxValidator? TxValidator { get; set; }
+        public TxValidator? TxValidator { get; set; }
```

Comment:
I would like this to remain an interface. Any reason not to?

Reply:
I had to change it to a concrete class since during initialization we might want to register validators for new `TxType`s:

https://github.com/NethermindEth/nethermind/blob/9437715fd3590891aa718891686b8aa5178a3d10/src/Nethermind/Nethermind.Optimism/InitializeBlockchainOptimism.cs#L77